scripts/js_dependencies.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# flake8: noqa
+import json
+import os
+import sys
+
+
+def collect_framework_deps():
+    return [tuple(split) for dep in os.listdir("rdrf/rdrf/static/vendor")
+            if len(split := dep.rsplit("-", 1)) == 2]
+
+
+def collect_js_deps():
+    return [tuple(split) for dep in os.listdir("rdrf/rdrf/static/js/vendor")
+            if len(split := dep.replace(".min.js", "").rsplit("-", 1)) == 2 and not dep.endswith(".map")]
+
+
+def deps_changed(deps):
+    """Check if the dependencies differ from those written to file"""
+    if not os.path.isfile("package.json"):
+        return True
+    else:
```

Comment:
You can drop the `else` and unindent the rest. One advantage of shortcutting functions as you did above is that you reduce the nesting.